Structure_Engine/Create/Offsets/Offset.cs

```diff
@@ -30,6 +34,14 @@ public static partial class Create
         /**** Public Methods                            ****/
         /***************************************************/
 
+        [Description("Creates a Offset defining offsets for Bar objects from its end Nodes to be applied in analysis packages.")]
+        [Input("startX", "Offset of the StartNode along the local x-axis, i.e. along the tangent.", typeof(Length))]
+        [Input("startY", "Offset of the StartNode along the local y-axis, i.e. along the axis perpendicular to the normal and tangent (generally minor axis).", typeof(Length))]
```

Comment:
The y-axis is typically the major axis.

Reply:
No, the y-axis is generally the minor. Rotation about the y-axis is major axis bending though.

Reply:
I have not really heard the y-axis referred to as the minor axis - my immediate though is that it was referring to the major/minor bending axis. 

Perhaps it could be a source of confusion because typically you would say Iy is the second moment of area about the major axis (y-axis) as in [3.2.5](https://www.steelforlifebluebook.co.uk/explanatory-notes/ec3-ukna/properties/).

Reply:
yeah, get what you mean, will get rid of it then.

Have generally been separating the transnational and rotational in my head at least (as well I think in the desc in general), like, Load acting along local z - major, Asz - major, Fz - major, and the rotational such as Iy, Wply, My - all major etc. 

And as this referred to transitional things, I matched it up that way, but if it is more confusing than helpful, then happy to get rid of it.